contextual_lenses/pfam_family_classification/pfam_one_hot_lenses_parallel.py

```diff
@@ -0,0 +1,223 @@
+"""Contextual lenses on Pfam family classification dataset (first 100 families)."""
```

Comment:
It would be good if we had a working doc for summarizing these results. I think the easiest thing would be to use google slides.